src/components/Layout/Layout.js

```diff
@@ -49,7 +53,23 @@ const Layout = ({ children }) => {
         <PrimaryNav />
       </div>
       <main className="l-main" id="main-content">
-        {children}
+        <div data-test="main-children">{children}</div>
+        {!cookies.release20_04 && (
+          <Notification
```

Comment:
Just out of curiosity why did you have to wrap children in a div? Were you not able to have `<Notification>` along side `{children}`?

Reply:
So I could target only the children with a test. The alternative was to include the notification text in the test which was too brittle.